### PR TITLE
Network Information API updates

### PR DESCRIPTION
**Pull request by jeffposnick**
R: @addyosmani @gauntface @beaufortfrancois 
CC: @jkarlin

This updates our existing Network Information API sample from the M38 era to include the new M48 functionality (https://www.chromestatus.com/feature/6338383617982464).

A live version of the updates can be found at https://jeffy.info/samples/network-information/, but as with the initial sample, you'll need an Android/iOS/Chrome OS device for the output to be meaningful.

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
LGTM :+1:

File: network-information/index.html [hunk 1]
```diff
@@ -1,95 +1,33 @@
...
+<p>
+  This demo illustrates the use of the
+  <a href="http://w3c.github.io/netinfo/">Network Information API</a>
+  to retrieve the current network type and monitor for network type changes.
+</p>
+
+<p>
+  The Network Information API is
+  <a href="https://code.google.com/p/chromium/issues/detail?id=368358#c18">not currently available</a>
+  on all platforms. For best results, try this demo using Chrome on Android, iOS, or ChromeOS.
```
Comment (jkarlin) [inline network-information/index.html]: The iOS bit isn't true. We don't support net info on ios unfortunately. Otherwise LGTM.

Comment (jeffposnick) [inline network-information/index.html]: Thanks for pointing that out!

**Comment by gauntface** [comment]
Breaks on browsers that don't support it.

**Comment by jeffposnick** [comment]
See my comment at https://github.com/GoogleChrome/samples/pull/258#issuecomment-159922809 

Is that not sufficient?

**Pull request merged**